umm-lib/test/cmr/umm/test/iso_smap/granule.clj

```diff
@@ -79,6 +80,8 @@
       ;; trim :orbital-model-name in each :orbit-calculated-spatial-domains
       (util/update-in-each [:orbit-calculated-spatial-domains]
                            #(assoc % :orbital-model-name (string/trim (:orbital-model-name %))))
+      ;; Remove crid-ids and feature-ids from umm-lib granule since they are not supported in echo10.
+      (update-in [:data-granule] dissoc :crid-ids :feature-ids)
```

Comment:
```suggestion
      (update-in [:data-granule] assoc :crid-ids nil :feature-ids nil)
```

Reply:
Like the `Suggested change` feature.

Reply:
Can't do it this way. when data-granule is nil, I will have an extra nil values there.